src/schemas/types/schema.ts

```diff
@@ -0,0 +1,55 @@
+import { createTuple } from '@/utilities'
+
+export const { values: schemaTypes, isValue: isSchemaType } = createTuple(['string', 'boolean', 'integer', 'number', 'array', 'object'])
+export type SchemaType = typeof schemaTypes[number]
+
+export const { values: schemaStringFormat, isValue: isSchemaStringFormat } = createTuple([
+  'date',
+  'date-time',
+  // 'time-delta',
+  'regex',
+  // 'email',
+  // 'json-string',
+  'password',
+])
```

Comment:
@znicholasbrown the commented out values are one's we had in the previous types that I don't know if we can actually produce from a pydantic type. Hoping you can help me validate that.

Reply:
Hm those came from Pydantic docs somewhere but I can't find it quickly. Let's start without; we've got clear paths to extend functionality everywhere so it probably makes more sense to focus on real primitives for now instead of ones that are infrequently used/unclear

Reply:
Agreed, and worse case the user will get a text input.